components/molecule/accordion/src/Tab/index.js

```diff
@@ -92,15 +94,20 @@ Tab.propTypes = {
   /**
    * Initial collapsed state
    */
-  isOpen: PropTypes.bool
+  isOpen: PropTypes.bool,
+  /**
+   * Add gap between tabs
+   */
+  enableTabGap: PropTypes.bool
 }
 
 Tab.defaultProps = {
   isOpen: false,
   maxHeight: MAX_HEIGHT,
   onToggle: () => {},
   withScrollVisible: false,
-  withTransition: true
+  withTransition: true,
+  enableTabGap: false
```

Comment:
To follow the other properties naming I think it may be better name"withGap" and the class "--withGap"

Reply:
done